jenkins/job-configs/kubernetes-jenkins-pull/fejta-pull.yaml

```diff
@@ -42,11 +42,10 @@
         - ansicolor:
             colormap: xterm
     builders:
-        - activate-gce-service-account
-        - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/test-infra/jenkins/upload-to-gcs.sh"
         - shell: |
-            git clone https://github.com/kubernetes/test-infra
-            './test-infra/{cmd}'
+            # TODO(fejta): change this back to kubernetes/test-infra -b master
+            git clone https://github.com/fejta/test-infra -b fejta
+            './test-infra/jenkins/builder.sh' '{cmd}'
```

Comment:
the name "builder" is also misleading, given that we have a `hack/jenkins/build.sh` in the main repo.